server/api/cvrs.py

```diff
@@ -1081,6 +1109,16 @@ def process_cvr_file(
 ):
     jurisdiction = Jurisdiction.query.get(jurisdiction_id)
 
+    temporary_directories_to_be_removed = []
+
+    def flag_temporary_directory_for_removal(path: str):
+        temporary_directories_to_be_removed.append(path)
```

Comment:
This control flow around the temp dir feels a bit zig-zaggy. What if we created the temp dir here and passed it down into the functions that need it, then cleaned it up at the end?

Reply:
@jonahkagan Updated in https://github.com/votingworks/arlo/pull/1726/commits/32574634564b1ccdb8f37b540970e8abf9927f88 and re-ran my manual test to verify that 1) I can still successfully upload OC's CVRs and 2) temp directory creation and cleanup is working as expected under the hood